src/SFA.DAS.Commitments.Api.Client/EmployerCommitmentApi.cs

```diff
@@ -90,6 +90,13 @@ public EmployerCommitmentApi(HttpClient client, ICommitmentsApiClientConfigurati
             await _commitmentHelper.PatchCommitment(url, submission);
         }
 
+        public async Task<IList<Apprenticeship>> GetActiveApprenticeshipsForUln(long employerAccountId, string uln)
```

Comment:
I can't see any tests coverage for this method. Other methods have covering tests in EmployerCommitmentApiTests